### ISSUE TITLE
Unable to boot from live USB

### ISSUE THREAD

**Issue by NathanHeath** [feren-OS/Bug-Reporting-Center#150]
**Package Name or Section of Feren OS where this bug resides**

This is with the live usb in the KDE version, 64 bit iso I downloaded July 15 2020.

**Describe the bug**

I was unable to boot from the live USB.

**Steps to Reproduce**

1. Flashed the iso onto a usb using both mint's usb writer and Rufus on windows. Both had the same result. The files were present on the usb but my computer was not able to boot from it.

When I attempt to boot I get a message "Failed to set MokListRT: Out of Resources"

My computer is a Dell Optiplex 790. Most linux distros I have attempted to boot were able to do so with no issue.

**Additional context**

I was able to install grub2 onto the usb from my linux computer. After that I was able to boot using the live usb.

**Comment by feren** [comment]
To write Feren OS to a USB Stick, you should follow the instructions on the Feren OS User Guide.
https://feren-os-user-guide.readthedocs.io/en/latest/writetousb.html

**Comment by NathanHeath** [comment]
I created a new live usb using etcher and on my Optiplex 790 it still doesn't boot and I still see the "Failed to set MokListRT: Out of Resources" message. However on my laptop, a dell e6440 it boots up just fine. I made yet another live usb using Rufus from windows and had the same result. On my laptop it boots as expected but on my desktop it doesn't. I do not understand why but I am going to try to track it down. 

This may be a problem more related to my hardware than your iso. However, this is the first time I have seen this error with any linux distro.
